### DISCUSSION TITLE
ai-generated-images-discourage-me-from-reading-your-blog/

### DISCUSSION THREAD

**Discussion by giscus** [nelsonfigueroa/nelson.cloud#19] [Announcements]
# ai-generated-images-discourage-me-from-reading-your-blog/

If you’re willing to use AI-generataed images, how do I know the text isn’t AI-generated?

https://nelson.cloud/ai-generated-images-discourage-me-from-reading-your-blog/

**Comment by MxPoesu** [comment]
Couldn't be truer

**Comment by rasheedmhd** [comment]
Agreed. I don't know what they think, that it makes them vogue?
I mean what happened to free stock photos?

**Comment by ufalsalman** [comment]
Totally agree with that. Even some shitty photos taken with some cheap mid-range phone or camera feels more human.

**Reply by nelsonfigueroa** [reply]
for real though

**Comment by jibal** [comment]
Hysteria.

**Comment by jasonpilbin** [comment]
It all started with spellcheck. Now we are doomed.
